.github/ansible/production/nest.yaml

```
@@ -7,80 +7,27 @@
-          mode: '0755'
-
-      - name: Copy backend makefile
-        ansible.builtin.copy:
-          src: '{{ github_workspace }}/backend/Makefile'
-          dest: ~/backend/Makefile
-          mode: '0644'
+      - name: Sync Makefile structure
+        ansible.builtin.synchronize:
+          src: '{{ github_workspace }}/'
+          dest: '~/'
+          recursive: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)